src/main/java/sirius/biz/storage/layer3/uplink/sftp/SFTPUplink.java

```diff
@@ -135,7 +135,6 @@ private MutableVirtualFile wrap(VirtualFile parent, SftpClient.DirEntry file, St
               .withDeleteHandler(this::deleteHandler)
               .withCanProvideInputStream(this::isExistingFile)
               .withInputStreamSupplier(this::inputStreamSupplier)
-              .withCanProvideOutputStream(this::isExistingFile)
```

Comment:
could you check all other Uplinks as well - just to be sure...

Reply:
Found one more, please check if this has to be removed.